client/src/legacy/CMSMain.EditForm.js

```diff
@@ -232,7 +232,7 @@ $.entwine('ss', function($){
 
       holder.next().filter('.listbox')[instant ? 'hide' : 'slideUp'](() => {
         holder.removeClass('field--merge-below');
-      });
+      }).css('overflow','visible'); // jquery adding overflow:hidden in hide, this will break listbox display
```

Comment:
After testing, I think this still needs work, if there are text below the hidden field, it'll overlap the textbox as it appears and slides down...